config/vufind/EDS.ini

```diff
@@ -403,3 +403,15 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
```

Comment:
I am seeing the same 'Short' vs 'Long' behavior as you, and I appreciate the detail below about where each kind is extracted from the result.  From looking through the EDS API docs previously, I doubt there is any official explanation of the two data elements.  But if you happen to have found one somewhere, it would be great to link it here for further info.

Reply:
That's great thanks @maccabeelevine 

Unfortunately we haven't found any official docs that we can link. The information in the EDS.ini that we have added is just what we have learnt through debugging the EDS API.

That being the case I will mark this thread as resolved.

Reply:
As expected sadly, but thanks for checking!

Reply:
Not a problem :)